src/storage/simple.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, save_folder: Union[str, Path], save_freq: int = 1):
         self.save_path.mkdir(parents=True)
         self.save_freq = save_freq
 
-    def save(self, epoch: int, modules: dict[str, nn.Module], metric: dict[str, float]):
+    def save(self, epoch: int, modules: Dict[str, nn.Module], _metric: Dict[str, float]):
```

Comment:
why metric is changed to _metric?

Reply:
Initially it was to supress the linter warning about the unused parameter, but to think about it, it's better to have a comment disabling that warning instead of the variable change, will fix.